server/pulp/server/db/model/__init__.py

```diff
@@ -307,3 +312,132 @@ def post_save(cls, sender, document, **kwargs):
 
 
 signals.post_save.connect(TaskStatus.post_save, sender=TaskStatus)
+
+
+class ContentUnit(Document):
+    """
+    The base class for all content units.
+
+    All unit inheriting from this class should override the unit_type_id and _ns to ensure
```

Comment:
Do all subclasses need to override unit_type_id and _ns? If they all must then s/should/must/